### DISCUSSION TITLE
Looking for ideas to calculate RANK in a BudibaseDB table

### DISCUSSION THREAD

**Discussion by dgagliardi** [Budibase/budibase#6008] [Help]
Hi! I have a budibaseDB table with a column called Rank. In a table I want to scan another table and calculate the rank for each person. It's pretty simple in excel and google sheets. Looking for ideas on how to solve in a budibaseDB table.

Thanks
D

**Comment by melohagan** [comment]
Hey @dgagliardi 

Have you had a look at formulas?  https://docs.budibase.com/docs/formula
